apps/notifications/style.scss

```diff
@@ -44,6 +44,29 @@
 		font-size: $font-title-medium;
 	}
 
+	.wp-block-button__link {
+		color: white;
+		font-weight: bold;
+		font-size: 1rem;
+		line-height: 1;
+		background-color: #1279be;
+		/* stylelint-disable-next-line scales/radii */
+		border-radius: 5px;
+		padding: 16px 24px;
+
+		&:hover {
+			background-color: #303030;
+			cursor: pointer;
+		}
+	}
```

Comment:
Same as below, once D50508-code lands we likely won't need most of these styles, given that most of them are already defined in the [button block styles](https://github.com/WordPress/gutenberg/blob/a1f60f701f2fc48cf1bdddad78084805c90b2ded/packages/block-library/src/button/style.scss#L5-L34).

Reply:
I tested without these styles. I'm not sure that's true.

I double-patched my sandbox by applying:

D51180-code and D50508-code and checked out this branch.

This is what I see on the UI:

<img width="318" alt="Screen Shot 2020-10-15 at 4 49 18 PM" src="https://user-images.githubusercontent.com/66652282/96184394-6eea4780-0f06-11eb-8b2c-431ca7eef5a0.png">

DOM:
<img width="274" alt="Screen Shot 2020-10-15 at 4 49 32 PM" src="https://user-images.githubusercontent.com/66652282/96184414-7578bf00-0f06-11eb-94b0-acaad23883ca.png">

You can see the classes passed through, but there's no styles.

Perhaps there's something I'm not understanding, or your changes are missing something?

These buttons also render as `a` tags nested in `divs`, and not actual `button`s, for what its worth.

Reply:
> I double-patched my sandbox by applying: D51180-code and D50508-code

Did they get applied cleanly? They both modify the same file, so maybe some changes are lost due to conflicts.

The lack of styles in particular seems to be caused by the lack of the D50508-code's change that moves `$this->ranges[] = $object;` just before the `break` statement.

Reply:
> Did they get applied cleanly?

That was it:

<img width="1112" alt="Screen Shot 2020-10-16 at 10 59 59" src="https://user-images.githubusercontent.com/1233880/96238451-c63bf680-0f9e-11eb-9697-3d1856a36973.png">

In order to solve these conflicts you'd need to `arc path` one diff and apply the changes of the other manually (copy/paste from the diff). Once the changes from both diffs are applied in the sandbox correctly, I see some inline styles are there:

<img width="773" alt="Screen Shot 2020-10-16 at 11 09 50" src="https://user-images.githubusercontent.com/1233880/96239533-20898700-0fa0-11eb-9ff7-84c36fd1b1bb.png">

Reply:
> These buttons also render as a tags nested in divs, and not actual buttons, for what its worth.

This is expected. Gutenberg saves button blocks as links:

https://github.com/WordPress/gutenberg/blob/dfdc623bcb359f0e8593587741877285d23d143f/packages/block-library/src/button/save.js#L38

<img width="1272" alt="Screen Shot 2020-10-16 at 11 12 25" src="https://user-images.githubusercontent.com/1233880/96239877-837b1e00-0fa0-11eb-9358-274a9e7d2aed.png">

Reply:
@mmtr you're probably right -- I never saw a conflict. I removed the styles. Much better this way anyway as we preserve single source of truth!